recipe/meta.yaml

```diff
@@ -49,7 +49,6 @@ outputs:
       string: rapidsai_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
       ignore_run_exports_from:
         - {{ compiler('cuda') }}  # [(cuda_compiler_version or "").startswith("11")]
-        - nccl                    # [linux and cuda_compiler != "None"]
```

Comment:
Based on Philip's testing, we need `nccl` in the install environment otherwise we see library loader errors. So this keeps `nccl`'s `run_exports` to ensure `nccl` will be added (and properly constrained) in `run`. That way it is installed in the user's environment